runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -155,14 +151,6 @@ public void render(Graphics2D graphics, final OverlayLayer layer)
 				overlayPosition = overlay.getPreferredPosition();
 			}
 
-			if (overlayPosition == OverlayPosition.ABOVE_CHATBOX_RIGHT && !isResizeable)
```

Comment:
This looked important. Why was it removed?

Reply:
Actually probably doing both is better idea

Reply:
Updated